phing/tasks/setup.xml

```diff
@@ -229,6 +229,10 @@
         <drush command="config-import" assume="yes" alias="${drush.alias}" passthru="false">
           <param>${cm.core.key}</param>
         </drush>
+        <!-- Sometimes config-import has to be run twice to catch everything. Can be removed once the core configuration system is more stable. -->
```

Comment:
@danepowell maybe add a URL referencing the bug (in core) that's in question here...